docs/source/models/photons.rst

```diff
@@ -0,0 +1,50 @@
+Photons
+=======
+
+Radiation reaction and (hard) photons: why and when are they needed.
+Models we implemented and verified:
+
+* Landau-Lifschitz Model (semi-classical)
```

Comment:
Yes, we can make a separate section for it but should still link it *from* here since nearly all comparisons will need it :)